mlflow/java/client/src/main/java/org/mlflow/tracking/Page.java

```diff
@@ -0,0 +1,35 @@
+package org.mlflow.tracking;
+
+import java.util.List;
+import java.util.Optional;
+
+interface Page<E> {
+
+  /**
+   * @return Returns the number of elements in this page.
```

Comment:
nit: I think the convention is not to add the extra `Returns`

more like
`@return The number of elements in this page`.